src/store/middlewares/account.js

```diff
@@ -18,6 +26,14 @@ const updateAccountData = (store, action) => {
     transactions,
     account,
   }));
+
+  if (action.data.passphrase) {
```

Comment:
It's not clear to me why this needs to be added. Maybe adding a comment would help. 
And if it's needed only for autologin, could be also explicitly expressed by adding `&& shouldAutoLogIn(getAutoLogInData()` to the condition